lib/hexo/index.js

```diff
@@ -212,6 +212,16 @@ class Hexo extends EventEmitter {
   }
 
   init() {
+    // Load only the relevant plugins in hexo clean
+    if (this.env.cmd === 'clean') {
```

Comment:
Initially this was `cmd.startsWith('c')`, however in light of https://github.com/hexojs/hexo/pull/4388, it must now match 'clean'.

https://github.com/hexojs/hexo/pull/4388 (i.e. `alias[this.env.cmd] === 'clean'`) could not be used here because console plugins (consequently their command aliases) have not been registered at this point.

This means using alias like `hexo c` will not benefit from this shortcut. To retain `hexo c` alias (to avoid breaking change), `clean` console plugin is still registered in "plugins/console.js".